src/middleware/genesys/README.md

```diff
@@ -17,53 +17,56 @@ This reference implementation supports the core features of a Genesys Cloud inte
 
-1. In Genesys Cloud, [create a widget](https://help.mypurecloud.com/articles/create-a-widget-for-web-chat/) to enable users to chat with agents. Make a note of the widget ID.
+1. In Genesys Cloud, [create a widget](https://help.mypurecloud.com/articles/create-a-widget-for-web-chat/) to enable users to chat with agents. Make a note of the generated `Deployment Key`. Follow the instructions for either a `1.1` or `1.0` widget.
 
-1. Update [`purecloudSetup.ts`](../../serviceDesks/genesys/purecloudSetup.ts) to populate it with your information.
+1. Make sure there is an active `Queue` by selecting your profile bubble on the left sidebar and clicking on `Activate Queues`. Check that there is at least one queue under `Active`. If not, either activate an `Available` queue on the right, or [create a queue](https://help.mypurecloud.com/articles/create-queues/). In any case, take note of the name of the `Active` queue you would like to use.
+    - **Creating a queue:**
+      - Name must not include any spaces. This queue name should be unique, and ideally describe the purpose/grouping of the queue. Make sure you click Save.
+      - Click on the queue that you just created and add yourself through the `Members` tab (you do not have to click save).
+      - Any newly created Queues will be `Active` by default.
 
-  - `ORGANIZATION_ID`: Your Genesys Cloud organization ID. You can find this ID in the [Genesys Cloud settings](https://help.mypurecloud.com/faq/how-do-i-find-my-organization-id/).
-  - `DEPLOYMENT_ID`: The ID of the widget you created in the previous step.
+1. Set up OAuth Client server:
+    1. If you haven't done so already, [set up a Genesys Cloud OAuth client](https://help.mypurecloud.com/articles/create-an-oauth-client/).
 
-1. Go to the project root directory and edit the `.env` file. Update the `SERVICE_DESK_CLASS` variable to `GenesysServiceDesk`.
-
-1. From the project root directory, run `npm run dev`. If you've linked everything to your Genesys Cloud account correctly, you should be able to connect to an agent in Genesys Cloud.
+    1. In the `src/middleware/genesys` subdirectory, rename or copy `.env-sample` to `.env`.
 
-To configure advanced functionality such as authenticated chat and agent availability status, follow these steps:
+    1. In the `.env` file, update the values of the `GENESYS_CLIENT_ID` and `GENESYS_CLIENT_SECRET` to the credentials from your Genesys OAuth client.
 
-1. If you haven't done so already, [set up a Genesys Cloud OAuth client](https://help.mypurecloud.com/articles/create-an-oauth-client/).
+    1. From the `src/middleware/genesys` directory, run `npm install`.
 
-1. In the `example-server` subdirectory, rename or copy `.env-sample` to `.env`.
+    1. From the `src/middleware/genesys` directory, run `npm start`. This starts a server on port 3000 of your local machine.
```

Comment:
minor comment - change `This starts a server on port 3000 of your local machine` to `This starts a server on port 3000 on your local machine`